docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -182,7 +209,24 @@ Type | Key | Value | Default value | Description
 &#xfeff;                                                     |                             | relu                        |                         | Relu function
 &#xfeff;                                                     |                             | softmax                     |                         | Softmax function
 &#xfeff;                                                     | return_sequences            | (boolean)                   | false                   | Return only the last output if true, else return full output
+&#xfeff;                                                     | bidirectional               | (boolean)                   | false                   | Option for bidirectional
```

Comment:
This should be removed. we are not going to release the bidirectional LSTM in M1.